src/common/styles/common.css

```diff
@@ -1,169 +1,169 @@
-.margin-bottom-xxs {
+.margin-bottom-xxsmall {
   margin-bottom: 2px;
 }
-.margin-bottom-xs {
+.margin-bottom-xsmall {
```

Comment:
These are common css classes we can use to arrange/size our elements. Usage is fairly straightforward. 
In another file I'll cover a little more on how to create these for our use. 
We should try to use these standard sizing/spacing whenever possible.
*requires classnames*
``` <div className={cx('margin-left-small)}```